src/components/timeline/TrackContextMenu.js

```diff
@@ -150,8 +150,6 @@ class TimelineTrackContextMenu extends PureComponent<Props> {
         data={{ trackIndex }}
         onClick={this._toggleGlobalTrackVisibility}
         attributes={{
-          // The following id is used to select this item in tests.
-          'data-test-id': 'global-track-' + trackIndex,
```

Comment:
Because `react-testing-library` makes it easier, the tests are now matching the textContent instead of some artificial attribute. So `data-test-id` isn't needed anymore.